hmpps-prisoner-search-indexer/Running.md

```diff
@@ -1,125 +1,40 @@
-docker compose up localstack hmpps-auth prison-api restricted-patients hmpps-incentives-api hpsi-db
-```
-
-Once localstack has started then, in another terminal, run the following command to start prisoner indexer too:
-```shell
-docker compose up hmpps-prisoner-search-indexer --detach
-```
-  To check that it has all started correctly use `docker ps`.
+The docker-compose.yml file is provided as reference but the dependencies of dependencies are not shown and are subject
+to change. To run against T3 dependencies the environment variables as shown in the helm_deploy/values-dev.yaml file
+for paths can be used. Personal client credentials for the T3 services can be obtained from the HMPPS Auth team 
+with the appropriate roles.
 
-#### Punning prisoner search indexer in IntelliJ or on the command line
-To start up localstack and other dependencies with prisoner offender search running in IntelliJ:
+## Running prisoner search indexer in IntelliJ or on the command line
+To run the indexer from the command line:
 ```shell
-docker compose up --scale hmpps-prisoner-search-indexer=0
-```
-To then run prisoner offender search from the command line:
-```
 SPRING_PROFILES_ACTIVE=dev ./gradlew bootRun
 ```
 Alternatively create a Spring Boot run configuration with active profile of `dev` and main class
-`uk.gov.justice.digital.hmpps.prisonersearch.search.HmppsPrisonerSearch`.
+`uk.gov.justice.digital.hmpps.prisonersearch.indexer.HmppsPrisonerSearchIndexer`.
 
-#### Running the tests
+## Running the tests
 If just running the tests then the following will just start localstack as the other dependencies are mocked out:
-
-```shell
-docker compose -f docker-compose-localstack-tests.yml up
-```
-Then the following command will run all the tests:
-```shell
-./gradlew test
-```
-
-#### Deleting localstack data between runs
-Since localstack persists data between runs it may be necessary to delete the localstack temporary data:
-
-Mac
-```shell
-rm -rf $TMPDIR/data
-```
-Linux
-```shell
-sudo rm -rf /tmp/localstack
-```
-Docker Desktop for Windows (started using `docker-compose-windows.yml`)
-```shell
-docker volume rm -f prisoner-offender-search_localstack-vol
-```
-
-*Please note the above will not work on a Mac using Docker Desktop since the Docker network host mode is not supported on a Mac*
-
-On Mac it is recommended to run all components *except* prisoner-offender-search (see below). Then run prisoner-offender-search outside of docker using gradle:
-
 ```shell
-SPRING_PROFILES_ACTIVE=dev ./gradlew bootRun
+docker compose -f docker-compose-test.yml pull && docker compose -f docker-compose-test.yml up -d
 ```
 
-### When running locally you can add some prisoners into Elastic with the following:-
-
-#### Get a token
-```shell
-TOKEN=$(curl --location --request POST "http://localhost:8090/auth/oauth/token?grant_type=client_credentials" --header "Authorization: Basic $(echo -n prisoner-offender-search-client:clientsecret | base64)" |  jq -r .access_token)
```

Comment:
this is all far too difficult to do locally since it relies on Prison API having data.  Would have to use the indexer to grab the data and then search to view it.  I've never done it locally and therefore think it doesn't deserve to be included